### PR TITLE
[React-native] Adding showSoftInputOnFocus

### PR DESCRIPTION
**Pull request by Andarius**
Adding showSoftInputOnFocus to TextInputAndroidProps (https://reactnative.dev/docs/textinput#showsoftinputonfocus)

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -1225,6 +1225,11 @@ export interface TextInputAndroidProps {
      * Vertically align text when `multiline` is set to true
      */
     textAlignVertical?: 'auto' | 'top' | 'bottom' | 'center';
+    
+    /**
+     * Prevent the soft keyboard from showing when the field is focused if false. The default value is true.
```
Comment (ca057) [inline types/react-native/index.d.ts:1230]: Shall we use the documentation as on the React Native website you linked? It sounds a bit easier to understand :)

**Review by ca057 — APPROVED** [review]

**Review by a-tarasyuk — CHANGES_REQUESTED** [review]
It seems tests failed due to lint error, could you fix it?

> ERROR: 1228:1  no-trailing-whitespace  trailing whitespace

File: types/react-native/index.d.ts (hunk 1 above)
Comment (Andarius) [reply types/react-native/index.d.ts:1230]: sure, changing that now

**Review by a-tarasyuk — APPROVED** [review]

**Comment by alloy** [comment]
Thanks!

**Pull request merged**